### DISCUSSION TITLE
What's the deal with the 52 week lead times on these chips?

### DISCUSSION THREAD

**Discussion by lerouxb** [SpenceKonde/megaTinyCore#378] [Q&A]
The QFN variants seem to be out of stock everywhere and Microchip's talking about a year long lead time. Doesn't Microchip manufacture their own chips? I thought the shortage largely affected TSMC. Or are they using their capacity for more popular lines? Can't find anything online to explain why these chips specifically are out of stock.

This sucks because I wanted to use the new 1 series chips in a product and now I'm having second thoughts :(

Anyway - thought you might have some insider info.

**Comment by SpenceKonde** [comment]
There's a letter to customers in a press release from their CEO (basically apologizing for the shortage, but also telling customers something to the effect of "You canceled your orders in the spring, what did you expect us to do?" ) from like late summer 2020., Microchip's capacity is extremely constrained. IIRC they are more impacted than industry average. There is just a sudden and large imbalance between supply and demand for fab time right now. *(maybe if semiconductor companies invested more of their earnings in manufacturing capacity, and put less of it towards many-billion-dollar buyouts of their peers, they would be better able to meet demand!). I know Microchip/Atmel is/was big in automotive; I am almost certain that it's Microchip's fault that big name car makers have idled factories, despite there still being customer demand,. because they can't get their hands of the chips they need for them).

My guess is the tiny-suitable production capacity is getting pointed at the 2-series instead . Either that, or someone is buying every batch they they can get their hands on and bought up months worth of production.

**Comment by SpenceKonde** [comment]
You're not able to use the 2-series? They're available in the same 3mm x 3mm QFN

**Reply by lerouxb** [reply]
I "need" the DAC.

**Reply by lerouxb** [reply]
I suppose I could go for filtered PWM if I really had to but the DAC was a large draw for me to this line in the first place.

**Comment by SpenceKonde** [comment]
Also,. holy crap @ lead times. When I checked last month it was only out to september This kind of stuff does make me wish they would just jack up the per-unit price, mint money, and stuff the kitty with money to keep the wheels spinning in bad times , or invest to boost production, or develop new technology...

**Comment by SpenceKonde** [comment]
Filtered PWM sucks. 

Modern AVRs (DA and tiny, with a DAC, in QFN, and in stock at digikey:
https://www.digikey.com/en/products/filter/embedded-microcontrollers/685?s=N4IgjCBcoEwJwWiAxlAZgQwDYGcCmANCAPZQDaIMAbAAwDsVArCEQBxUDMMzRMHcMVjRABdIgAcALlBABlSQCcAlgDsA5iAC%2BRRjSpRQqSJlyES5EBzB0w3UROmQ5i1Ru0hGYOAZTps%2BIlJICg4aRlYwHksw1gAWDhZo8JgYRNDwiLSwgSj0uFjvIlDaKjpE2N0YSKyqWLoyojrWOhhhIr1GDmYxECkZeWV1LSIwUdSkIxMA82CQKjAODlj7XsdnQbcR%2BERDP1NAizB4plZEo-4ws8ZdWOWR665UkdYXxjvKai8oz9ZGVJ6%2Bk4Bq5huA4KxvBM9tMghRhAC1sChu4ALTjXaQRQAVzMsI8ok07mYSHEUDAElJkG47n0SCUABMZCiwDQIA4ZIlJABPcR4GQYHCoQlAA


If you ever wonder why the commercial product you took apart to gawk at before throwing away has a whole bunch of unpopulated pads. It's times like these, when they've gotta design the same thing three ways because they have no idea which vendor will be able to deliver. I've seen three implementations of a whole subsystem on a board before - like a 4x4 inch section of PCB 2/3rds of which was unpopulated, Cases where the main chip has several available options are even more commo n.

**Reply by lerouxb** [reply]
Thanks! I should have enough space on my board and don't need much flash or many pins, so in theory I could fit lots of different ones. Gonna try the multiple footprints thing.

**Comment by freemovers** [comment]
A couple weeks ago I was looking for any ATtiny*17, and according to the Microchip website the earliest deliveries were for 2022. A week later about 1500 of the ATtiny1617 were available on Digikey. I know that it is difficult to plan around a 52 weeks lead-time, but looks like that is more an expected catch-up time, and they are still releasing smaller batches (or Digikey found some uncounted for).

**Comment by grandaspanna** [comment]
Just ordered some 1626 QFN directly from Microchip which should hopefully be a drop-in for my 1616 board. Ordered two days ago and are apparently in local DHL facility already...

**Reply by grandaspanna** [reply]
So the 1626 works nicely. Not sure if it's just me, but had to adjust some constants in the event subsystem for the code to compile, but otherwise, pretty much a drop-in for my board. Given it's also cheaper than a 555, I may deploy it for blinking LEDs... Looks like there's still some stock at microchipdirect.
